mm2src/mm2_main/src/lp_swap/taker_swap_v2.rs

```diff
@@ -944,8 +945,8 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             },
         };
 
-        let total_payment_value =
-            &(&state_machine.taker_volume + &state_machine.dex_fee.total_spend_amount()) + &state_machine.taker_premium;
+        let total_payment_value = &(&state_machine.taker_volume + &state_machine.dex_fee().total_spend_amount())
+            + &state_machine.taker_premium;
```

Comment:
Related to this thread https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112#discussion_r1980070997

Reply:
I refactored the check of dex_fee for dust in [473930c](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2112/commits/473930c380ca01df0b64d17fac2a45400337bac3). Is this comment still active (could you plz add more info then)?